frameworks/PHP/sw-fw-less/sw-fw-less.dockerfile

```diff
@@ -0,0 +1,84 @@
+FROM php:7.1
+
+MAINTAINER luoxiaojun1992 <luoxiaojun1992@sina.cn>
+
+# Version
+ENV PHPREDIS_VERSION 4.0.0
+ENV HIREDIS_VERSION 0.13.3
+ENV SWOOLE_VERSION v4.4.0
+
+# Timezone
+RUN /bin/cp /usr/share/zoneinfo/Asia/Shanghai /etc/localtime \
+    && echo 'Asia/Shanghai' > /etc/timezone
+
+# Libs
+RUN apt-get update \
+    && apt-get install -y \
+        curl wget git zip unzip less vim procps lsof tcpdump htop openssl \
+        libz-dev \
+        libssl-dev \
+        libnghttp2-dev \
+        libpcre3-dev \
+        libjpeg-dev \
+        libpng-dev \
+        libfreetype6-dev
+
+# Composer
+RUN curl -sS https://getcomposer.org/installer | php \
+    && mv composer.phar /usr/local/bin/composer \
+    && composer self-update --clean-backups
+
+# PDO extension
+RUN docker-php-ext-install pdo_mysql
+
+# Bcmath extension
+RUN docker-php-ext-install bcmath
+
+# Sockets extension
+RUN docker-php-ext-install sockets
+
+# Redis extension
+RUN wget http://pecl.php.net/get/redis-${PHPREDIS_VERSION}.tgz -O /tmp/redis.tar.tgz \
```

Comment:
You shouldn't be using redis for anything, right? You're not doing the cached queries test. Please remove.